crates/bevy_scene/src/bundle.rs

```diff
@@ -0,0 +1,66 @@
+use bevy_asset::Handle;
+use bevy_ecs::{
+    bundle::Bundle,
+    change_detection::ResMut,
+    entity::Entity,
+    prelude::{Changed, Component},
+    system::Query,
+};
+use bevy_transform::components::{GlobalTransform, Transform};
+
+use crate::{DynamicScene, InstanceId, Scene, SceneSpawner};
+
+#[derive(Default, Component)]
+pub struct HasSceneInstance(Option<InstanceId>);
```

Comment:
This type is pub, but the field is unreadable. We should probably pub the field IMO to make it more useful, or at least write a getter.

Reply:
implemented `Deref` and `DerefMut` for it